jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -55,6 +55,9 @@
 import com.ibm.gpu.spi.GPUAssist;
 import com.ibm.gpu.spi.GPUAssistHolder;
 /*[ENDIF] Sidecar19-SE */
+/*[IF !Sidecar18-SE-OpenJ9 | PLATFORM-mz31 | PLATFORM-mz64]*/
```

Comment:
Parentheses are advised: `(!Sidecar18-SE-OpenJ9) | PLATFORM-mz31 | PLATFORM-mz64`, or reorder
so precedence can't be an issue: `PLATFORM-mz31 | PLATFORM-mz64 | !Sidecar18-SE-OpenJ9`.